spanner/spanner_snippets/spanner/spanner_create_instance_with_autoscaling_config.go

```diff
@@ -61,7 +61,8 @@ func createInstanceWithAutoscalingConfig(w io.Writer, projectID, instanceID stri
 					StorageUtilizationPercent:         95,
 				},
 			},
-			Labels: map[string]string{"cloud_spanner_samples": "true"},
+			Labels:  map[string]string{"cloud_spanner_samples": "true"},
+			Edition: instancepb.Instance_ENTERPRISE_PLUS,
```

Comment:
nit: does this need to be ENTERPRISE_PLUS or is ENTERPRISE sufficient for autoscaling here? wondering if we can set the base use.